src/objects/deck/deck.scss

```diff
@@ -60,3 +60,30 @@
 .o-deck--align-start {
   align-items: start;
 }
+
+/**
+ * Block alignment adjustments
+ *
+ * 1. Set inline padding to match `o-container__fill-pad`.
+ *    We're using the container tokens here, but ideally we'd make a
+ *    token for the whole `fluid-clamp()` formula, but our token system
+ *    doesn't support using Sass formulas that way. This should be okay,
+ *    since we're still matching the container's tokens.
+ * 2. Remove padding once `alignwide` is no longer full-bleed.
+ */
+
+.o-deck.alignfull,
+.o-deck.alignwide {
+  padding-inline: fluid.fluid-clamp(
+    size.$padding-container-horizontal-min,
+    size.$padding-container-horizontal-max,
+    breakpoint.$s,
+    breakpoint.$xl
+  ); // 1
+}
+
+.o-deck.alignwide {
+  @media (width >= breakpoint.$l) {
+    padding-inline: 0; // 2
+  }
+}
```

Comment:
Alternatively, you could override the typical `margin-*` behavior with the same that the `o-container__fill` and `__fill-pad` elements use. If there's a specificity issue there, you could apply these styles to the `wp-block-*` classes in our vendor CSS (to better associate this with the conflicting styles).

Reply:
I'm afraid I don't follow this comment at all. Probably just me being tired. Can we chat with voices tomorrow morning about this?

Reply:
Re-reading my comment, I think I misunderstood why this style existed. Looking at it with fresh eyes this morning (and no migraine), I now understand that this isn't really 1:1 with `o-container__fill-pad` and that this padding override exists to fill a more unique need.

I do think in the future we may find that these `.alignwide` and `.alignfull` customizations make more sense alongside other WordPress styles (rather than with the components), but I don't feel strongly enough about that to argue more coherently. I think we're fine as-is.

I'm available to chat with voices sometime this afternoon (I have a few other tasks to take care of before a long morning meeting), but I don't think this should block your progress: I'm good with the current solution.

Reply:
👍🏻